src/EditorFeatures/Test2/IntelliSense/ModelTests.vb

```diff
@@ -111,7 +111,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
 
             modelComputation.ChainTaskAndNotifyControllerWhenFinished(Function(m, c)
                                                                           checkpoint1.Release()
-                                                                          checkpoint2.Task.Wait()
+                                                                          checkpoint2.Task.Wait(c)
                                                                           c.ThrowIfCancellationRequested()
```

Comment:
📝 The semantics here changed. This method no longer waits for `checkpoint2.Task` to complete before proceeding.